include/vecops.hxx

```diff
@@ -56,6 +56,9 @@ const Vector3D Grad(const Field3D &f, CELL_LOC outloc = CELL_DEFAULT);
 /// @param[in] outloc  The cell location where the result is desired
 ///
 const Vector3D Grad_perp(const Field3D &f, CELL_LOC outloc = CELL_DEFAULT);
+inline const Vector3D Grad_perp(const Field3D &f, CELL_LOC outloc, DIFF_METHOD method){
```

Comment:
Aside from `DIFF_METHOD` not doing anything here, none of the other vecops take `DIFF_METHOD` (apart from `Div`, weirdly).

We shouldn't add this unless we're prepared to make this useful

Reply:
If #1384 makes it in then this will touch quite a few of the difops headers in order to try to make things a bit more consistent (https://github.com/boutproject/BOUT-dev/pull/1384/files#diff-d4f46bfa8ae0ce6e06e2b165c3b810c2) as well as supporting an alternative way of specifying the method.